bcml/cli.py

```diff
@@ -0,0 +1,185 @@
+                meta={
+                    "name": meta["name"],
+                    "version": meta["version"],
+                    "desc": meta["description"],
+                    "platform": "wiiu"
+                    if get_settings("wiiu")
+                    else "switch",
+                },
+            )
+    elif args.export:
+        # --export
+        install.export(args.export)
+
+    elif args.standalone:
+        # --standalone
+        bnp = args.standalone
+        with TempModContext():
+            install.install_mod(
+                bnp,
+                merge_now=True,
+                options={"options": {"texts": {"all_langs": True}}, "disable": []},
+            )
+            
+            out = f"{bnp.stem}.zip"
+            if out:
+                output = Path(out)
+                install.export(output, standalone=True)
+    
+    elif args.update:
+        # --update
+        update_file = args.update;
+        for i in get_installed_mods(disabled=True):
+            name = i.name.replace(" ", "").lower()
+            arg = update_file.name.rsplit(".")[0].replace(" ", "").lower()
+            if name == arg:
+                mod = i;
+
+        if (mod.path / "options.json").exists():
+            options = json.loads((mod.path / "options.json").read_text());
+        else:
+            options = {};
+
+        shutil.rmtree(mod.path)
+        with Pool(maxtasksperchild=500) as pool:
+            new_mod = install.install_mod(
+                Path(update_file),
+                insert_priority=mod.priority,
+                options=options,
+                pool=pool,
+                updated=True,
+            );
+
+        install.refresh_master_export()
+
+    elif args.remerge:
+        # --remerge
+        install.refresh_merges()
+        install.refresh_master_export()
+
+    elif args.install:
+        # --install
+        install.install_mod(
+            mod=args.install, 
+            insert_priority=args.priority if args.priority else 0, 
+            merge_now=True
+        )
+        install.refresh_master_export()
+
+    elif args.uninstall:
+        # --uninstall
+        for mod in util.get_installed_mods(disabled=True):
+            name = mod.name.replace(" ", "").lower()
+            arg = args.uninstall.replace(" ", "").lower()
+            if name == arg:
+                install.uninstall_mod(mod=mod)
+    
+    elif args.uninstall_all:
+        for mod in get_installed_mods(disabled=True):
+            install.uninstall(mod=mod, wait_merge=True)
+            install.refresh_merges()
+            install.refresh_master_export()
+
+    elif args.backup:
+        # --backup
+        install.create_backup(args.backup)
+
+    elif args.restore:
+        # --restore
+        backup = list((util.get_storage_dir() / "backups").glob(f"{args.restore}*"))
+        if len(backup) <= 0:
+            raise ValueError("The backup does not exists!")
```

Comment:
Now I'm just being pedantic, but... "The backup does not exist"

Also, for clarity's sake, I tend to f-string the "error" into the message. I've noticed that people tend to think the problem is not their fault, so if the error message is "the backup does not exist" then they'll go looking for the backup to make sure it exists, instead of looking at their argument to see if it's misspelled.

I think something like `f"No backup named {args.restore} exists!"` would work. If you wanted to go above and beyond, you could generate a suggestion. ("Did you mean {other_similar_backup_name}?")

Reply:
Maybe in the future I'll implement some sort of fuzzy string matching, but for now I'll just leave it at the error you suggested. It does sound better